tests/python/frontend/darknet/test_forward.py

```diff
@@ -59,6 +70,10 @@ def _get_tvm_output(net, data, build_dtype="float32", states=None):
     """Compute TVM output"""
     dtype = "float32"
     mod, params = relay.frontend.from_darknet(net, data.shape, dtype)
+    # verify that from_darknet creates a valid, parsable relay program
+    mod = relay.transform.InferType()(mod)
+    astext(mod)
```

Comment:
remove astext

Reply:
I think we're missing each other. The bug is because the IRModule generated using `from_darknet` is invalid to the parser, so parsing from the text representation back to an IRModule using `tvm.parser.parse` fails. 

astext() checks that works `tvm.parser.parse(mod.astext())`

Reply:
this is a general problem that variable names aren't normalized in the frontend code

Reply:
ahh, I see. then should we put it in a try block? otherwise ppl might be confused when we get trapped in astext

Reply:
I'm not quite sure what you mean? This is the test file, so a try block would prevent a test from failing?

Reply:
yeah, I was saying we probably want to explicitly catch the failure and assert for unit test. Something like the following,

```
test_pass = True
try:
   astext(mod)
except:
   print("fail")
  test_pass = False

assert test_pass
```

But anyway, it is not mandatory since this is unit test